### PR TITLE
[Docs]Add some more details to docs

### PR DESCRIPTION
**Pull request by HIT-cwh**
Easy to use.

### CODE REVIEW

File: docs/en/test.md [hunk 1]
```diff
@@ -8,10 +8,20 @@ To test nas method, you can use following command
...
 - `MUTABLE_CFG_PATH`: Path of `mutable_cfg`. `mutable_cfg` represents **config for mutable of the subnet searched out**, used to specify different subnets for testing. An example for `mutable_cfg` can be found [here](https://github.com/open-mmlab/mmrazor/blob/master/configs/nas/spos/SPOS_SHUFFLENETV2_330M_IN1k_PAPER.yaml).
 
 The usage of optional arguments are the same as corresponding tasks like mmclassification, mmdetection and mmsegmentation.
 
+For example,
+
+<pre>
+python tools/mmcls/test_mmcls.py \
+  configs/nas/spos/spos_subnet_shufflenetv2_8xb128_in1k.py \
+  <em>your_retraining_checkpoint_path</em> \
```
Comment (wutongshenqiu) [inline docs/en/test.md:21]: NAS algorithms like once-for-all or BigNAS may not need to retrain subnet.
```suggestion
  <em>your_subnet_checkpoint_path</em> \
```

File: docs/en/train.md [hunk 2]
```diff
@@ -33,9 +50,18 @@ python tools/${task}/train_${task}.py ${CONFIG_FILE} --cfg-options algorithm.mut
...
+python ./tools/mmcls/train_mmcls.py \
+  configs/nas/spos/spos_subnet_shufflenetv2_8xb128_in1k.py \
+  --work-dir $WORK_DIR \
+  --cfg-options algorithm.mutable_cfg=configs/nas/spos/SPOS_SHUFFLENETV2_330M_IN1k_PAPER.yaml
+</pre>
+
 ## Pruning
 
-Pruning has four steps, including **supernet pre-training**, **search for subnet on the trained supernet**, **subnet retraining** and **split checkpoint**. The commands of the first two steps are similar to NAS, except that we need to use `CONFIG_FILE` of Pruning here. The commands of the two other steps are as follows.
+Pruning has three steps, including **supernet pre-training**, **search for subnet on the trained supernet** and **subnet retraining**. The commands of the first two steps are similar to NAS, except that we need to use `CONFIG_FILE` of Pruning here. The commands of the **subnet retraining** is as follows.
```
Comment (wutongshenqiu) [inline docs/en/train.md:64]:
```suggestion
Pruning has three steps, including **supernet pre-training**, **search for subnet on the trained supernet** and **subnet retraining**. The commands of the first two steps are similar to NAS, except that we need to use `CONFIG_FILE` of Pruning here. commands of the **subnet retraining** are as follows.
```

**Review by wutongshenqiu — CHANGES_REQUESTED** [review]

File: docs/en/test.md (hunk 1 above)
Comment (HIT-cwh) [reply docs/en/test.md:21]: Fixed, thx!

**Review by wutongshenqiu — APPROVED** [review]

File: docs/en/test.md [hunk 3]
```diff
@@ -8,10 +8,20 @@ To test nas method, you can use following command
...
 - `MUTABLE_CFG_PATH`: Path of `mutable_cfg`. `mutable_cfg` represents **config for mutable of the subnet searched out**, used to specify different subnets for testing. An example for `mutable_cfg` can be found [here](https://github.com/open-mmlab/mmrazor/blob/master/configs/nas/spos/SPOS_SHUFFLENETV2_330M_IN1k_PAPER.yaml).
 
 The usage of optional arguments are the same as corresponding tasks like mmclassification, mmdetection and mmsegmentation.
 
+For example,
+
+```bash
+python tools/mmcls/test_mmcls.py \
+  configs/nas/spos/spos_subnet_shufflenetv2_8xb128_in1k.py \
+  <em>your_subnet_checkpoint_path</em> \
```
Comment (wutongshenqiu) [inline docs/en/test.md:21]: ```bash``` and `<em> </em>` seems to have conflicts

**Pull request merged**